src/Misc/atoms/MediaBox.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState, useCallback, VideoHTMLAttributes } from 'react';
 import styled, { css } from 'styled-components';
 import Spinner from '../../Indicators/Spinner';
 import { IMediaType } from '../../index';
-import Icon from '../../Icons/Icon';
+import { ReactComponent as LargeNoImage } from '../../svg/Large-No-image.svg';
```

Comment:
Please change this to **NoImageBig.svg**

LargeNoImage  --> **NoImage**

Reply:
done